src/query/api/v1/handler/prometheus/native/read_common.go

```diff
@@ -86,36 +86,47 @@ func read(
 		}
 	}()
 
-	firstElement := false
-	var numSteps, numSeries int
+	var (
+		numSteps   int
+		numSeries  int
+		exhaustive bool
```

Comment:
Hm, should this not start out as `exhaustive = true`?

Reply:
Good call, makes sense for the empty result case